tests/integ/modin/conftest.py

```diff
@@ -437,7 +437,7 @@ def loc_snowpark_pandas_input_map():
         "series[bool]_col": (
             pd.Series(
                 [True, True, True, False, False, True, True],
-                index=pandas.Index(["A", "B", "x", "D", "F", "E", "C"]),
+                index=["A", "B", "x", "D", "F", "E", "C"],
```

Comment:
This is a pandas Index still right?

Reply:
No, this will create a modin index for the series. That is what we want right?